src/main/java/pages/ProductPage.java

```diff
@@ -28,9 +28,9 @@ public void clickOnLogotip() {
         logotip.click();
     }
 
-    public void clickAddToCartButton() {
+    public void clickAddToCartButton() throws InterruptedException {
         buttonAddToCart.click();
-        waitVisibilityOfElement(60,shoppingCart);
+        Thread.sleep(1000);
```

Comment:
Remove "sleep" and add "explicitWait" (it relates on all "sleep" methods in your repo)